client/packages/electron/README.md

```diff
@@ -0,0 +1,43 @@
+## Electron Client
+
+When server is running as an `offline` sync server on local network there are a couple of problems that's a bit hard to solve with configurations: 
+* Discovery (where is server, and what if it's local IP changes)
+* SSL (self signed certs and browsers are not friendly)
+
+It seems like the most robust way to solve this issue is with a client app:
+* Discover server with mDNS
+* Trust certificates coming from discovered server
+* Display web content served by remote server (no need to upgrade client when server upgrades)
+* Allow for future native functionality (potentialy can use another bundler when time comes to implement native functionality, like Tauri)
+
+### Running
+
+Make sure front end is built and run server (front end needs to be built because server, not webpack, will be serving front end to the client)
```

Comment:
```suggestion
Make sure front end is built and run the server (the front end needs to be built because the server, not webpack, will be serving front end to the client)
```